build-parent/pom.xml

```diff
@@ -546,27 +546,54 @@
                     </configuration>
                 </plugin>
                 <plugin>
-                    <groupId>io.fabric8</groupId>
+                    <!--
+                        NB: using a temporary fork of the docker-maven-plugin for faster local build.
+                        This can be removed once https://github.com/fabric8io/docker-maven-plugin/issues/1804
+                        is incorporated into the io.fabric8 plugin directly.
+                      -->
+                    <groupId>org.technologybrewery.fabric8</groupId>
                     <artifactId>docker-maven-plugin</artifactId>
                     <version>${version.fabric8.docker.maven.plugin}</version>
                     <extensions>true</extensions>
+                    <executions>
+                        <execution>
+                            <id>default-push</id>
+                            <phase>deploy</phase>
+                            <goals>
+                                <goal>push</goal>
+                            </goals>
+                            <configuration>
+                                <images>
+                                    <image>
+                                        <build>
+                                            <buildx>
```

Comment:
I (Informational): configure the `docker-maven-plugin` to use the buildx `docker-container` driver at deploy time and to/from caching to improve build speed.